card.py

```diff
@@ -1,7 +1,8 @@
+from constants import CHANGE_COLOR_CARD_NUM
 
 
 class Card:
-    def __init__(self, num=0, color="white", change_color=False):
+    def __init__(self, num=CHANGE_COLOR_CARD_NUM, color="change color", change_color=False):
```

Comment:
Why are you using zero for change color? if it meaningless you can use null, then it is very explicit like "It doesn't have a number"

Reply:
Why are you using the "change color" value for color? I couldn't find that here :) 
https://7esl.com/colour-vocabulary/

That kind of patch make the code complex, and force you to do more patch 
if it meaningless you can use null, then it is very explicit like "It doesn't have a color"

Reply:
change_color is a verb and isn't a noun that defines a state. 
suggestion: is_change_color